PROTOCOL.md

```diff
@@ -1308,7 +1308,7 @@ This section documents additional requirements that writers must follow in order
   - Add / Remove actions with the same `(path, DV)` tuple.
   - More than one Metadata action
   - More than one protocol action
-  - More than one SetTransaction with the same `appId`
+  - More than one Transaction Identifiers with the same `appId`
```

Comment:
Follows the naming here, right?
https://github.com/delta-io/delta/blob/master/PROTOCOL.md#transaction-identifiers

Reply:
Yes